ui/components/app/transaction-detail/transaction-detail.component.js

```diff
@@ -92,7 +92,13 @@ export default function TransactionDetail({
 }
 
 TransactionDetail.propTypes = {
+  /**
+   * Show item content for transaction detail. Array of TransactionDetailItem components
+   */
   rows: PropTypes.arrayOf(TransactionDetailItem).isRequired,
+  /**
+   * onClick handler for the Edit link
+   */
   onEdit: PropTypes.func,
   userAcknowledgedGasMissing: PropTypes.bool.isRequired,
```

Comment:
Needs `userAcknowledgedGasMissing` propType description. Maybe @jpuri can provide an explanation for what the prop does

Reply:
resolved